rott/vgatext.c

```diff
@@ -0,0 +1,198 @@
+/*
+MIT License
+
+Copyright (c) 2024 erysdren (it/she/they)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of
+this software and associated documentation files (the "Software"), to deal in
+the Software without restriction, including without limitation the rights to
+use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+of the Software, and to permit persons to whom the Software is furnished to do
+so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+#include <SDL2/SDL.h>
+
+#include "vga8x16.h"
+
+#define BLINK_HZ ((1000 / 70) * 16)
+
+static const Uint8 palette[16][3] = {
```

Comment:
Which palette is this and where is it from?

Reply:
https://github.com/fabiangreffrath/woof/commit/19d3ab0ae8a8396f1784f9de448b5aa8922dd7d0#diff-a3b338dd6c7567493b9ddb1f7b0600d22d78f30a764cee4b0ffcac898732e940R88-R106

Reply:
Ah, okay. So, if this is the standard EGA palette, it might make sense to call it like that or at least leave a comment what it is.